packages/config/src/layer2s/eclipse.ts

```diff
@@ -0,0 +1,43 @@
+import { EthereumAddress, ProjectId } from '@l2beat/shared-pure'
+
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import { CONTRACTS, TECHNOLOGY, UPCOMING_RISK_VIEW } from './common'
+import { Layer2 } from './types'
+
+const discovery = new ProjectDiscovery('eclipse')
```

Comment:
You're not using discovery since your project is still upcoming. You can safely delete this. Otherwise the build is going to fail on you.